src/apis/chat/README.md

```diff
@@ -33,6 +33,30 @@ const { chat, auth } = Chat;
 </script>
 ```
 
+## Quick NodeJS example
```

Comment:
Just to be consistent with a bunch of our other libraries (we use a template on modules such as `nodejs-storage`) let's make this heading:

`## Using the client library`